proxy/wpr.py

```diff
@@ -0,0 +1,72 @@
+import os
+import signal
+import subprocess
+import sys
+
+from proxy.mitmproxy import MITMProxy404
+
+dirname = os.path.dirname(__file__)
+
+
+class Wpr(object):
```

Comment:
Can we rename this to `WebPageReplay` to make it more clear?

Reply:
Done